src/components/pod/pod.test.js

```diff
@@ -13,6 +55,234 @@ const PodWithSoftDelete = () => (
 );
 
 context("Testing Pod component", () => {
+  describe("should render Pod component", () => {
+    it.each([
+      [true, "0px none rgb(0, 0, 0)"],
+      [false, "0px none rgb(0, 0, 0)"],
+    ])(
+      "should check when border is %s for Pod component",
```

Comment:
Think it would be better to say "should check when border is %s the border value is %s for Pod component "